docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -0,0 +1,728 @@
+---
+assignees:
+- bprashanth
+- enisoc
+- erictune
+- foxish
+- janetkuo
+- kow3ns
+- smarterclayton
+---
+
+{% capture overview %}
+This tutorial provides an introduction to the 
```

Comment:
It's not really an introduction to the StatefulSet concept; that's what the concept doc is for.

This is an introduction to the basics of using StatefulSet.